### ISSUE TITLE
Overlapping scrollbar with close button in "Markdown Cheatsheet"

### ISSUE THREAD

**Issue by heldderarbeit** [toolstack/Folio#111]
If you hover over the scrollbar it is rendered on top of the X button, there should be some space between them.

![Untitled](https://github.com/toolstack/Folio/assets/4840355/eb8fa5e2-1b5e-44f0-a5b8-0170b2babb4e)

Installed version is 24.10 from Flathub

**Comment by toolstack** [comment]
That's a function of gnome that I don't think I can change, I guess I could add a buffer to the top of the window so that the cheat sheet appeared below the close button, but that seems like a waste of space.

**Comment by toolstack** [comment]
Closing as wontfix.

**Comment by heldderarbeit** [comment]
If you are open to suggestions regarding this - How about a regular headerbar? Here's how it would look like before / after.

![image](https://github.com/toolstack/Folio/assets/4840355/eb685681-e807-4395-a2c8-80c389ddc1a1)

Or another comparison with a different version, maybe moving up the first header to save space:

![image](https://github.com/toolstack/Folio/assets/4840355/600631e0-5450-4751-af65-12fbd9be5812)

In that case the "Headings" line would be roughly on the same position.


Implementation-wise I found out that it's relatively easy: nesting the GtkMarkdownView in an Adw.StatusPage and replacing Adw.Window with Gtk.Window is enough.

```
using Gtk 4.0;
using Adw 1;

template $FolioMarkdownCheatsheet : Gtk.Window {

  default-width: 500;
  default-height: 440;
  titlebar: Adw.HeaderBar {};

  ShortcutController {
  …
  }

  Adw.StatusPage {
      css-name: _("status_page");

      $GtkMarkdownView text_view {

      tab-width: 4;
      …
      }
  }

}

```

That would bring 3 advantages:
1) Scrollbar won't overlap anymore with close button
2) it's not possible anymore to resize the window like this (resizing is completely blocked like in the keyboard shortcuts window):
![image](https://github.com/toolstack/Folio/assets/4840355/6d7233ef-6a95-4913-a79d-9996c838f4cf)

[Screencast from 2024-05-07 20-35-10.webm](https://github.com/toolstack/Folio/assets/4840355/141f7ab8-b7af-453c-8572-61247d526b9f)

3) less code compared to now, for example ScrolledWindow is not needed anymore since scrolling is done automatically by Adw's StatusPage

If there is interest I could send a merge request for one of the headerbar versions (but really, there's not more code than I have listed here).

I would be very glad to see this realized since I very much like the general aesthetics of the app (it feels like it almost could belong to GNOME Circle and I would encourage you to apply your app for submission) but feel they are tarnished by the behaviour I described here. Therefore I stepped in to give some additional information and possible resolutions, despite having no prior experience with GTK elements beforehand.

It's a small change, but in my opinion would result in a much more professional look.
